nav2_costmap_2d/test/simple_driving_test.xml

```diff
@@ -0,0 +1,14 @@
+<launch>
```

Comment:
Same comment as above - not sure this test should be added unless it's going to be made to work in ROS2.